packages/labs/scoped-registry-mixin/package.json

```diff
@@ -28,18 +28,118 @@
     "/scoped-registry-mixin.{d.ts,d.ts.map,js,js.map}"
   ],
   "scripts": {
-    "build": "npm run clean && npm run build:ts --build && rollup -c",
-    "build:watch": "rollup -c --watch",
-    "build:ts": "tsc --build && treemirror development . '**/*.d.ts{,.map}'",
-    "build:ts:watch": "tsc --build --watch",
-    "clean": "rm -rf {scoped-registry-mixin}.{js,js.map,d.ts} development/ test/ *.tsbuildinfo",
-    "dev": "scripts/dev.sh",
-    "test": "npm run test:dev && npm run test:prod",
-    "test:dev": "MODE=dev node ../../tests/run-web-tests.js \"development/**/*_test.js\" --config ../../tests/web-test-runner.config.js",
-    "test:prod": "MODE=prod node ../../tests/run-web-tests.js \"development/**/*_test.js\" --config ../../tests/web-test-runner.config.js",
-    "test:watch": "npm run test:dev -- --watch",
-    "checksize": "rollup -c --environment=CHECKSIZE",
-    "regen-package-lock": "rm -rf node_modules package-lock.json; npm install"
+    "start": "wireit",
+    "build": "wireit",
+    "build:ts": "wireit",
+    "build:ts:types": "wireit",
+    "build:rollup": "wireit",
+    "test": "wireit",
+    "test:dev": "wireit",
+    "test:prod": "wireit",
+    "checksize": "wireit"
+  },
+  "wireit": {
+    "start": {
+      "command": "web-dev-server -- --node-resolve --watch --preserve-sym-links --open demo/index.html",
```

Comment:
Unrelated to this PR and non blocking. I could not run this command due to `web-dev-server` command not found. Since it's not in the `devDependencies` I assume this needs to be global.

Reply:
`scoped-registry-mixin` depends on `@web/test-runner-mocha`, which is the same as what everything else has. It really doesn't work for you after a top-level bootstrap?